priprava/jazyky-nastroje/instalace-python.md

```diff
@@ -0,0 +1,21 @@
+## Instalace jazyka Python
+
+### Windows
+Pokud máte počítač s Windows, následujte tyto kroky:
+
+1. Ujistěte se, že máte aktualizovaný systém. To zajistíte nejjednodušeji tak, že svůj počítač restartujete. 
+1. Stáhněte si instalaci jazyka Python z [tohoto odkazu](https://www.python.org/ftp/python/3.9.10/python-3.9.10-amd64.exe). Stažený soubor spusťte. Rozběhne se průvodce instalací.
```

Comment:
Raději bych nechala konkrétní verzi Pythonu a cca 1x za rok to updatovala. Na úvodní kurz by to nevadilo, ale mám špatnou zkušenost s `latest` v kombinaci s Windows a některými balíčky (matplotlib).

Reply:
Naprosto souhlasím, ale tenhle problém (kombinace Pythonu 3.10 a matplotlibu na windows) je stále aktuální, takže než se vyřeší, tak bych se držela radši verze, která je stabilní i pro balíčky, které používáme.